forester/src/epoch_manager.rs

```diff
@@ -29,77 +30,39 @@ use light_test_utils::indexer::{Indexer, MerkleProof, NewAddressProofWithContext
 use light_test_utils::rpc::rpc_connection::RpcConnection;
 use log::{debug, error, info, warn};
 use rand::Rng;
-use solana_account_decoder::UiAccountEncoding;
-use solana_client::nonblocking::pubsub_client::PubsubClient;
-use solana_client::rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig};
-use solana_sdk::commitment_config::CommitmentConfig;
 use solana_sdk::compute_budget::ComputeBudgetInstruction;
 use solana_sdk::instruction::Instruction;
 use solana_sdk::pubkey::Pubkey;
 use solana_sdk::signature::{Signature, Signer};
 use solana_sdk::transaction::Transaction;
 use std::collections::HashMap;
 use std::iter::Zip;
-use std::str::FromStr;
+use std::pin::Pin;
 use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::Arc;
-use std::time::{Duration, Instant};
-use std::{mem, thread};
-use tokio::runtime::Builder;
+use std::time::Duration;
 use tokio::sync::{mpsc, oneshot, Mutex, Semaphore};
-use tokio::time::sleep;
-
-type Result<T> = std::result::Result<T, ForesterError>;
-
-#[derive(Debug, Clone)]
-pub struct QueueItemData {
-    pub hash: [u8; 32],
-    pub index: usize,
-}
-
-pub async fn fetch_queue_item_data<R: RpcConnection>(
-    rpc: &mut R,
-    queue_pubkey: &Pubkey,
-) -> Result<Vec<QueueItemData>> {
-    debug!("Fetching queue data for {:?}", queue_pubkey);
-    let mut account = rpc.get_account(*queue_pubkey).await?.unwrap();
```

Comment:
Since you filed a PR removing unwraps recently, can we avoid them here too? :)

I know this error is very unlikely to happen, but maybe we could `expect()` at least